src/OrchardCore.Modules/OrchardCore.Contents/Views/Content.SummaryAdmin.cshtml

```diff
@@ -57,15 +71,3 @@
 {
     <div class="col primary">@await DisplayAsync(Model.Content)</div>
 }
-
-<script asp-name="removeEmptyActionsMenu" at="Foot" type="text/javascript">
```

Comment:
This was not working anyway. I can see the console error. (loads after jQuery)

Reply:
Good I find this too hacky.